pkg/config/config.go

```diff
@@ -34,7 +38,13 @@ func Save(config *api.Config, cmd *cobra.Command) {
 	}
 	// Store only flags that have changed
```

Comment:
this comment doesn't make sense.  isn't this saving all the flags?  (which seems fine)